packages/trader/src/Modules/SmartChart/Components/Markers/marker.tsx

```diff
@@ -26,10 +26,11 @@ const ChartMarker = ({ marker_config, marker_content_props, is_bottom_widget_vis
         if (ref) {
             // NOTE: null price means vertical line.
             if (!marker_props.y) {
-                const margin = (is_bottom_widget_visible ? 115 : 0) + 24; // digit contracts have a widget at the bottom // height of line marker icon
+                const margin = 24; // height of line marker icon
 
                 ref.div.style.height = `calc(100% - ${margin}px)`;
-                ref.div.style.zIndex = '-1';
+            } else {
+                ref.div.style.zIndex = '1';
```

Comment:
@ahmadtaimoor-deriv are these margin and zIndex changes intended? please check

Reply:
@maryia-deriv Yes, this change is intended :)